### PR TITLE
#16 Create separate constants files.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**

### CODE REVIEW

File: src/utils/formFieldLabelConstants.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+export const EMAIL_TEXT = 'ई-मेल';
+
+export const ADDRESS_TEXT = 'पूरा पता';
+
+export const BUS_STOP_TEXT = 'बस स्टॉप (कृपया निकटतम बस स्टॉप चुनें)';
+
+export const SELECT_NEXT_LEVEL = 'आप क्या अध्ययन करना चाहते हैं ?';
+
+export const LINK_INFORMATION_OF_EDIT_INFORMATION = 'कृपिया अन्य जानकारी बदलने हेतु यहाँ';
```
Comment (mohitsaladiya) [inline src/utils/formFieldLabelConstants.jsx]: Change it to EDIT_INFORMATION_LINK_TEXT

File: src/utils/labelConstants.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+
+export const PREVIOUS_YEAR_LEVEL_LABEL = 'पूर्व में किये गए धार्मिक अध्ययन का विवरण';
+
+export const IS_OPT_IN_OR_OPT_OUT_2019_LABEL = '2019 के शिविर की स्वीकृति ?';
+
+export const ID_LABEL = 'आई.डी.';
+
+export const NAME_LABEL = 'नाम';
+
+export const FATHER_OR_MOTHER_NAME_LABEL = 'पिता / पति का नाम';
```
Comment (mohitsaladiya) [inline src/utils/labelConstants.jsx]: Please change it to FATHER_OR_HUSBAND_NAME_LABEL

File: src/utils/labelConstants.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+export const EDUCATION_LABEL = 'स्कूल शिक्षा';
+
+export const EMAIL_LABEL = 'ई-मेल';
+
+export const ADDRESS_LABEL = 'पूरा पता';
+
+export const BUS_STOP_LABEL = 'बस स्टॉप (कृपया निकटतम बस स्टॉप चुनें)';
+
+export const SELECT_NEXT_LABEL = 'आप क्या अध्ययन करना चाहते हैं ?';
```
Comment (mohitsaladiya) [inline src/utils/labelConstants.jsx]: Please change it to WHAT_YOU_WANT_TO_STUDY_LABEL

File: src/utils/labelConstants.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+export const MARK_ATTENDANCE_POPUP_HEADER_LABEL = 'कृपिया चयनित छात्रो की उपस्तिथि दर्ज करे ';
```
Comment (mohitsaladiya) [inline src/utils/labelConstants.jsx]: MARK_SELECTED_STUDENTS_ATTENDANCE_LABEL

File: src/utils/labelConstants.jsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+export const OPT_IN_OR_OPT_OUT_POPUP_HEADER_LABEL = 'कृपया चयनित छात्रों की शिविर के लिए हाँ या ना दर्ज़ करें';
```
Comment (mohitsaladiya) [inline src/utils/labelConstants.jsx]: OPT_IN_STATUS_FOR_SELECTED_STUDENTS_LABEL

File: src/utils/labelConstants.jsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+export const UPDATE_ID_CARD_STATUS_POPUP_HEADER_LABEL = 'कृपिया चयनित छात्रों के पहचान पत्र का print status अद्यतन करे';
```
Comment (mohitsaladiya) [inline src/utils/labelConstants.jsx]: ID_CARD_PRINT_STATUS_FOR_SELECTED_STUDENTS_LABEL

File: src/utils/messagesConstants.jsx [hunk 7]
```diff
@@ -0,0 +1,52 @@
+
+export const SELECTED_ZERE_COLUMNS_MESSAGE = 'आपने शून्य स्तंभों को चुना है इसलिए वहाँ जानकारी उपलब्ध नहीं है।';
```
Comment (mohitsaladiya) [inline src/utils/messagesConstants.jsx]: ZERE? 
It should be NO_COLUMNS_SELECTED_MESSAGE

File: src/utils/messagesConstants.jsx [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+export const ID_NOTE_MESSAGE = 'कृपया अपना ID और सीक्रेट कोड ध्यानपूर्वक नोट कर लेवे |';
+
+export const ID_CARD_SUGGESTION_MESSAGE = 'शीघ्र ही आपका ID Card आपके क्षेत्रीय संयोजक द्वारा भेजा जायेगा |';
+
+export const UPDATED_ID_CARD_STATUS_SUCCESS_MESSAGE = 'चयनित छात्रों के पहचान पत्र का print status सफलतापूवर्क अद्यतन कर दी गयी है|';
+
+export const UPDATED_ID_CARD_STATUS_FAILED_MESSAGE = 'चयनित छात्रों के पहचान पत्र का print status अद्यतन करने में असफल रहे|';
+
+export const UPLOAD_OPT_IN_FILE_SUCCESS_MESSAGE = ' छात्रों की optin file सफलतापूवर्क अपलोड कर दी गयी है|';
```
Comment (mohitsaladiya) [inline src/utils/messagesConstants.jsx]: OPT_IN_FILE_UPLOAD_SUCCESS_MESSAGE

File: src/utils/messagesConstants.jsx [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+export const UPLOAD_OPT_IN_FILE_FAILED_MESSAGE = 'छात्रों की optin file अपलोड पूर्णतः असफल रहा|';
```
Comment (mohitsaladiya) [inline src/utils/messagesConstants.jsx]: OPT_IN_FILE_UPLOAD_FAILURE_MESSAGE

File: src/utils/messagesConstants.jsx [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+export const UPLOAD_ATTENDANCE_FILE_SUCCESS_MESSAGE = 'छात्रों की उपस्तिथि फाइल सफलतापूवर्क अपलोड कर दी गयी है|';
```
Comment (mohitsaladiya) [inline src/utils/messagesConstants.jsx]: ATTENDANCE_FILE_UPLOAD_SUCCESS_MESSAGE

File: src/utils/messagesConstants.jsx [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+
+export const UPLOAD_ATTENDANCE_FILE_FAILED_MESSAGE = 'छात्रों की उपस्तिथि फाइल की अपलोड असफल रही|';
```
Comment (mohitsaladiya) [inline src/utils/messagesConstants.jsx]: ATTENDANCE_FILE_UPLOAD_FAILURE_MESSAGE

File: src/utils/messagesConstants.jsx [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+
+export const INVALID_EMAIL_MESSAGE = 'यह ई-मेल मान्य नहीं है ';
+
+export const INVALID_NAME_MESSAGE = 'अमान्य नाम';
+
+export const NAME_LESS_THAN_THREE_CHARACTERS_NOT_VALID_MESSAGE = 'नाम ३ अक्षर से कम का मान्य नहीं';
+
+export const FULL_ADDRESS_MESSAGE = 'कृपया पूरा पता देंवे!';
+
+export const THIS_INFORMATION_HELPFUL_TO_CONTACTING_MESSAGE = 'यह जानकारी आपसे संपर्क करने में उपयोगी है!';
```
Comment (mohitsaladiya) [inline src/utils/messagesConstants.jsx]: INFORMATION_HELPFUL_TO_CONTACT_MESSAGE

File: src/utils/textConstants.jsx [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+
+export const YES_TEXT = 'हाँ';
+
+export const NO_TEXT = 'ना';
+
+export const ID_NUMBER_TEXT = 'आई.डी. नं.';
+
+export const SECRET_CODE_TEXT = 'सीक्रेट कोड';
+
+export const LINK_INFORMATION_OF_EDIT_INFORMATION_TEXT = 'कृपिया अन्य जानकारी बदलने हेतु यहाँ';
```
Comment (mohitsaladiya) [inline src/utils/textConstants.jsx]: UPDATE_FURTHER_INFORMATION_TEXT

File: src/utils/textConstants.jsx [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+
+export const CLICK_TEXT = 'क्लिक करे|';
```
Comment (mohitsaladiya) [inline src/utils/textConstants.jsx]: CLICK_HERE_TEXT

File: src/components/MarkOptInOrOptOutButton.jsx [hunk 15]
```diff
@@ -125,7 +125,6 @@ class MarkOptInOrOptOutButton extends Component {
       return (
         <div className="success-block">
           <span>चयनित छात्रों की शिविर के लिए हाँ या ना सफलतापूवर्क दर्ज़ कर दी गयी है|</span>
```
Comment (mohitsaladiya) [inline src/components/MarkOptInOrOptOutButton.jsx]: import/use constants to display message here.

File: src/components/MarkSelectedStudentAttendance.jsx [hunk 16]
```diff
@@ -130,15 +130,15 @@ class MarkSelectedStudentAttendance extends Component {
     if (this.props.isMarkAttendanceSuccess) {
       return (
         <div className="success-block">
-          <span>चयनित छात्रो की उपस्तिथि सफलतापूवर्क दर्ज कर दी गयी है |</span>
+          <span>चयनित छात्रो की उपस्तिथि सफलतापूवर्क दर्ज कर दी गयी है|</span>
```
Comment (mohitsaladiya) [inline src/components/MarkSelectedStudentAttendance.jsx]: import/use constants to display message here.

File: src/components/MarkSelectedStudentAttendance.jsx [hunk 17]
```diff
@@ -130,15 +130,15 @@ class MarkSelectedStudentAttendance extends Component {
...
         </div>
       );
     } else if (!this.props.isMarkAttendanceSuccess && this.props.isMarkAttendanceFailed) {
       return (
         <div className="upload-message-wrapper">
           <div className="failure-block">
             <span>
-              चयनित छात्रों की उपस्तिथि दर्ज करने में असफल रहे |
+              चयनित छात्रों की उपस्तिथि दर्ज करने में असफल रहे|
```
Comment (mohitsaladiya) [inline src/components/MarkSelectedStudentAttendance.jsx]: import/use constants to display message here.

File: src/components/UpdateIdCardStatusSelectedStudents.jsx [hunk 18]
```diff
@@ -98,7 +98,7 @@ class UpdateIdCardStatusSelectedStudents extends Component {
         <div className="upload-message-wrapper">
           <div className="failure-block">
             <span>
-              चयनित छात्रों के पहचान पत्र का print status अद्यतन करने में असफल रहे |
+              चयनित छात्रों के पहचान पत्र का print status अद्यतन करने में असफल रहे|
```
Comment (mohitsaladiya) [inline src/components/UpdateIdCardStatusSelectedStudents.jsx]: import/use constants to display message here.

File: src/components/UploadOptInFile.jsx [hunk 19]
```diff
@@ -130,7 +130,7 @@ class UploadOptInFile extends Component {
         <div className="upload-message-wrapper">
           <div className="failure-block">
             <span>
-             छात्रों की optin file अपलोड पूर्णतः असफल रहा |
+             छात्रों की optin file अपलोड पूर्णतः असफल रहा|
```
Comment (mohitsaladiya) [inline src/components/UploadOptInFile.jsx]: import/use constants to display message here.

File: src/components/UploadStudentsAttendanceFile.jsx [hunk 20]
```diff
@@ -108,7 +108,6 @@ class UploadStudentsAttendanceFile extends Component {
       return (
         <div className="upload-message-wrapper">
           <div className="success-block">
-            {/* Upload students attendance file is success*/}
             छात्रों की उपस्तिथि फाइल सफलतापूवर्क अपलोड कर दी गयी है|
```
Comment (mohitsaladiya) [inline src/components/UploadStudentsAttendanceFile.jsx]: import/use constants to display message here.

File: src/components/UploadStudentsAttendanceFile.jsx [hunk 21]
```diff
@@ -119,7 +118,7 @@ class UploadStudentsAttendanceFile extends Component {
       return (
         <div className="upload-message-wrapper">
           <div className="failure-block">
-            छात्रों की उपस्तिथि फाइल की अपलोड असफल रही |
+            छात्रों की उपस्तिथि फाइल की अपलोड असफल रही|
```
Comment (mohitsaladiya) [inline src/components/UploadStudentsAttendanceFile.jsx]: import/use constants to display message here.

File: src/utils/formFieldLabelConstants.jsx [hunk 22]
```diff
@@ -0,0 +1,52 @@
...
+
+export const CLICK = 'क्लिक करे|';
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/utils/formFieldLabelConstants.jsx]: Done.

File: src/utils/labelConstants.jsx (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/utils/labelConstants.jsx]: Done.

File: src/utils/labelConstants.jsx (hunk 3 above)
Comment (BhagyashreeBhoyar) [reply src/utils/labelConstants.jsx]: Done.

File: src/utils/labelConstants.jsx (hunk 4 above)
Comment (BhagyashreeBhoyar) [reply src/utils/labelConstants.jsx]: Done.

File: src/utils/labelConstants.jsx (hunk 5 above)
Comment (BhagyashreeBhoyar) [reply src/utils/labelConstants.jsx]: Done.

File: src/utils/messagesConstants.jsx (hunk 10 above)
Comment (BhagyashreeBhoyar) [reply src/utils/messagesConstants.jsx]: Done.

File: src/utils/messagesConstants.jsx (hunk 11 above)
Comment (BhagyashreeBhoyar) [reply src/utils/messagesConstants.jsx]: Done.

File: src/utils/textConstants.jsx (hunk 13 above)
Comment (BhagyashreeBhoyar) [reply src/utils/textConstants.jsx]: Done.

File: src/utils/textConstants.jsx (hunk 14 above)
Comment (BhagyashreeBhoyar) [reply src/utils/textConstants.jsx]: Done.

File: src/components/MarkOptInOrOptOutButton.jsx (hunk 15 above)
Comment (BhagyashreeBhoyar) [reply src/components/MarkOptInOrOptOutButton.jsx]: I will done this in next PR.

File: src/components/MarkSelectedStudentAttendance.jsx (hunk 16 above)
Comment (BhagyashreeBhoyar) [reply src/components/MarkSelectedStudentAttendance.jsx]: I will done this in next PR.

File: src/components/MarkSelectedStudentAttendance.jsx (hunk 17 above)
Comment (BhagyashreeBhoyar) [reply src/components/MarkSelectedStudentAttendance.jsx]: I will done this in next PR.

File: src/components/UpdateIdCardStatusSelectedStudents.jsx (hunk 18 above)
Comment (BhagyashreeBhoyar) [reply src/components/UpdateIdCardStatusSelectedStudents.jsx]: I will done this in next PR.

File: src/components/UploadOptInFile.jsx (hunk 19 above)
Comment (BhagyashreeBhoyar) [reply src/components/UploadOptInFile.jsx]: I will done this in next PR.

File: src/components/UploadStudentsAttendanceFile.jsx (hunk 20 above)
Comment (BhagyashreeBhoyar) [reply src/components/UploadStudentsAttendanceFile.jsx]: I will done this in next PR.

File: src/components/UploadStudentsAttendanceFile.jsx (hunk 21 above)
Comment (BhagyashreeBhoyar) [reply src/components/UploadStudentsAttendanceFile.jsx]: I will done this in next PR.

File: src/utils/labelConstants.jsx (hunk 6 above)
Comment (BhagyashreeBhoyar) [reply src/utils/labelConstants.jsx]: Done.

File: src/utils/messagesConstants.jsx (hunk 7 above)
Comment (BhagyashreeBhoyar) [reply src/utils/messagesConstants.jsx]: Done.

File: src/utils/messagesConstants.jsx (hunk 8 above)
Comment (BhagyashreeBhoyar) [reply src/utils/messagesConstants.jsx]: Done.

File: src/utils/messagesConstants.jsx (hunk 9 above)
Comment (BhagyashreeBhoyar) [reply src/utils/messagesConstants.jsx]: Done.

**Pull request merged**